selenium_tests/class_roster/class_roster_permission_tests.py

```diff
@@ -35,7 +35,7 @@ def test_roles_access(self, user_id, given_access):
                 self.course_admin_dashboard_page
                     .class_roster_button_is_displayed(),
                 'User {} should see the class roster button on page but '
```

Comment:
I think you need to update this string as well to include the role (see e.g. https://github.com/Harvard-University-iCommons/canvas_manage_course/pull/18/files#diff-1f4bb4d36f243029f9a55633ae7b615dR41).